templates/static/js/src/demobox/components/LiveDemoBox/Select.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+export type DropdownInfo = {
+  query: string;
+  label: string;
+};
+
+export type DropdownProps = {
+  dropdownOptions?: [DropdownInfo];
+  dropdownLabel?: string;
+  disabled?: boolean;
+};
+
+const Select = ({dropdownOptions, dropdownLabel, disabled}: DropdownProps) => {
+  return (
+    <>
+      <label htmlFor="exampleSelect" style={{opacity: disabled === true ? 0.5 : 1}}>
+        {dropdownLabel}
+      </label>
+      <select name="exampleSelect" id="exampleSelect" disabled={disabled}>
```

Comment:
I just checked the `htmlFor` thing: [see here](https://developer.mozilla.org/en-US/docs/Web/API/HTMLLabelElement/htmlFor), I think it does refer to the `id`. We could make the `id` the `dropdownLabel` so it's different for each select?

Reply:
Ah, you're right. It's also possible to nest the `select` inside the `label` to avoid having to use `id` altogether. So like: 

```jsx
<label style={{opacity: disabled === true ? 0.5 : 1}}>
  {dropdownLabel}
  <select name="exampleSelect" disabled={disabled}>
</label>
```

More info [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label).